canframemodel.cpp

```diff
@@ -321,7 +321,7 @@ void CANFrameModel::recalcOverwrite()
         }
         else
         {
-            frame.timedelta = frame.timestamp - overWriteFrames[idAugmented].timestamp;
+            frame.timedelta = frame.timeStamp().microSeconds() - overWriteFrames[idAugmented].timeStamp().microSeconds();
```

Comment:
how would this work? `timestamp` before contained seconds+microseconds
`timeStamp().microSeconds()` is the **only** microseconds parts, so seconds are loose?

Reply:
```    class TimeStamp {
    public:
        Q_DECL_CONSTEXPR TimeStamp(qint64 s = 0, qint64 usec = 0) Q_DECL_NOTHROW
            : secs(s), usecs(usec) {}

        Q_DECL_CONSTEXPR static TimeStamp fromMicroSeconds(qint64 usec) Q_DECL_NOTHROW
        { return TimeStamp(usec / 1000000, usec % 1000000); }

        Q_DECL_CONSTEXPR qint64 seconds() const Q_DECL_NOTHROW { return secs; }
        Q_DECL_CONSTEXPR qint64 microSeconds() const Q_DECL_NOTHROW { return usecs; }

    private:
        qint64 secs;
        qint64 usecs;
    };
```